cypress/commons/services/stubbing.js

```diff
@@ -140,7 +151,9 @@ class Stubbing {
   getScenarios = () => this._getResources('scenarios');
   setScenarios = (newScenarios) => this._setResources('scenarios', newScenarios);
   addScenario = (newScenario) => this._addResource('scenarios', newScenario);
+  patchScenario = (scenarioId, scenarioPatch) => this._patchResourceById('scenarios', scenarioId, scenarioPatch);
   getScenarioById = (scenarioId) => this._getResourceById('scenarios', scenarioId);
+  getScenarioByName = (scenarioName) => this._getResourceIndexByName('scenarios', scenarioName);
```

Comment:
This should not work : you're returning the index of the resource instead of the resource itself

Reply:
This function should have been removed with previous modifications...